lib/catalog/martyrology.ts

```diff
@@ -30,6 +42,17 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       titles: [Title.Empress],
       dateOfDeath: '999-12-16',
     },
+    // src:
+    // - mr_fr_1982_ed2_coutances
+    // - https://www.diocese50.fr/les-paroisses-et-les-doyennes/notre-dame-coutances/les-saints-du-diocese/sainte-adeline-de-mortain-et-les-saints-de-savigny
+    // - https://fr.wikipedia.org/wiki/Adeline#:~:text=Adeline%20(%E2%80%A0%201125)%2C%20s%C5%93ur,tout%20l'Ordre%20de%20Savigny.
```

Comment:
At least in my case, the URL fragment of this URL does not lead to any specific place on the website. I presume you wanted to link to https://fr.wikipedia.org/wiki/Adeline#Saints_des_%C3%89glises_chr%C3%A9tiennes.

```suggestion
    // - https://fr.wikipedia.org/wiki/Adeline#Saints_des_%C3%89glises_chr%C3%A9tiennes
```

Reply:
I have just discovered [text fragments](https://developer.mozilla.org/en-US/docs/Web/URI/Fragment/Text_fragments) which did not work on the previous version of my web browser, therefore, I did not actually see the link to be working.

However, it is a fairly new feature and the document fragment is present on the referenced website, therefore, I still suggest to use the suggested source link.

Reply:
There was a small error in the URL I provided (an extra period at the end). But I prefer to keep my URL because it points directly to the passage that talks about Adeline and the saints of Savigny.

Reply:
Okay, let’s keep it, however, the final dot is not an issue, as it works even with it. :wink: